__tests__/app.test.js

```diff
@@ -113,3 +113,52 @@ describe.only("GET /api/articles", () => {
       });
   });
 });
+
+describe.only("GET /api/articles/:article_id/comments", () => {
+  test("200: Returns an array of comments objects", () => {
+    return request(app)
+      .get("/api/articles/1/comments")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.comments).toHaveLength(11);
+        body.comments.forEach((comment) => {
+          expect(comment).toMatchObject({
+            comment_id: expect.any(Number),
+            body: expect.any(String),
+            article_id: expect.any(Number),
+            author: expect.any(String),
+            votes: expect.any(Number),
+            created_at: expect.any(String),
+          });
+        });
+      });
+  });
+  test("200: Returns an array of comments objects with comment_id in ASC order", () => {
+    return request(app)
+      .get("/api/articles/3/comments")
+      .expect(200)
+      .then(({ body }) => {
+        body.comments.forEach((comment) => {
+          let prevCommentId = 0;
+          expect(comment.comment_id).toBeGreaterThan(prevCommentId);
+          prevCommentId = comment.comment_id;
+        });
+      });
+  });
+  test("404: Returns 'No comments found for this article'", () => {
```

Comment:
This test is a mixture of a couple so although you are thinking along the right lines...

If a valid and existing article id is passed but it has no comments then we should return the empty array "of the comments". This would not be an error.

If however the article id is valid but does not exist then we should return a 404 as the _article_ is not found.